code/__HELPERS/roundend.dm

```diff
@@ -246,8 +246,8 @@
 
 	CHECK_TICK
 
-	//check config blah blah
 	handle_hearts()
+	set_observer_default_invisibility(0)
```

Comment:
```suggestion
	set_observer_default_invisibility(0, "<span class='warning'>The round is over! You are now visible to the living.</span>")
```
Even though this mechanic will get figured out by players pretty quickly, some kind of feedback is always nice

Reply:
I'm amazed that there managed to be a review on a one line PR

Reply:
Scared?